src/canvas/Renderer.tsx

```diff
@@ -84,6 +84,42 @@ class Renderer implements IRenderer {
         return this._canvas;
     }
 
+    measurementInit = (): HTMLCanvasElement => {
```

Comment:
measurementInit does not need to return an element, it can be void.

Reply:
ah yes, I think that's a leftover of me trying things to solve some bugs. I will remove it.